test-app/src/widgets/Draggable.js

```diff
@@ -120,15 +123,19 @@ export default class Draggable extends createjs.Container {
     evt.stopPropagation();
     evt.preventDefault();
     this.accessible.grabbed = false;
+    let collided = false;
```

Comment:
Added the collided flag for clarity since targetId may be changed in the forEach now, so using _.isUndefined(this.targetId) doesn't necessarily mean that there was no collision